ampl-converter/src/main/java/com/powsybl/ampl/converter/AmplNetworkWriter.java

```diff
@@ -1707,6 +1772,9 @@ private void writeThreeWindingsTransformerCurrentLimits(TableFormatter formatter
 
     private void writeDanglingLineCurrentLimits(TableFormatter formatter) throws IOException {
         for (DanglingLine dl : network.getDanglingLines()) {
+            if (dl.isMerged()) {
+                continue;
+            }
```

Comment:
got it, it may give the impression that unmerged are separated from merged dangling lines. Maybe something like `getDanglingLines(Predicate<DanglingLine>)` would be more appropriate.